recipes/esmf/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ $(uname) == Linux ]]; then
+yum install -y gcc-gfortran
```

Comment:
Interesting, I would also be curious if we have `gfortran`. Might be worth using @jjhelmus' strategy here of adding these to the build scripts if it is still missing. ( https://github.com/conda-forge/docker-images/issues/1#issuecomment-197597726 )